src/query/storage/prom_converter.go

```diff
@@ -142,8 +146,17 @@ func toPromSequentially(
 			return PromResult{}, err
 		}
 
+		name := ""
+		nameTag, exists := tags.Get([]byte(model.MetricNameLabel))
```

Comment:
We could prob make this `MetricNameLabel` a const bytes too to avoid the cast

Reply:
There's a `promDefaultName` byte array for that, and I'll follow up in a new PR to address that.